### PR TITLE
Added the DropboxFinder to the manifest

### PR DESCRIPTION
**Pull request by ameenaltajer**
Hello guys,

I've been working on a Dropbox file finder plugin, here's the initial post:

https://github.com/Flow-Launcher/Flow.Launcher/discussions/1148

Hope you love this and add it to your manifest, Dropbox will make the application's status to "production" instead of "development" once more than 50 users use this, I'm sure a lot of people will find this useful as I'm using this on a daily basis now.

Best,
Ameen

### CODE REVIEW

**Comment by jjw24** [comment]
Hey this plugin will be really helpful. I did a fresh install and a quick test- the query box seems to be frozen after opening the oauth web page.

Also in settings it asks for the Dropbox path, does this mean if the file is found via the API it will be downloaded to the path set in settings?

**Comment by ameenaltajer** [comment]
Hello Jeremy,

1. This will stay frozen until you finish the OAuth acceptance and it will go away since there is an HTTP server waiting for the callback response, did you finish the steps and it's still frozen? If didn't please do.

2. This is your local Dropbox folder, this is used to navigate to the file locally on your machine, in the next update I'll cross-check the availability of the file on the machine, if not, I'll send the user to the preview page on Dropbox.com or setup a setting to download it automatically.

Best,
Ameen

**Comment by jjw24** [comment]
> 1. This will stay frozen until you finish the OAuth acceptance and it will go away since there is an HTTP server waiting for the callback response, did you finish the steps and it's still frozen? If you didn't please do.

Yeah ok that makes sense. I didn't finish the OAuth acceptance and I wonder if that's something that needs to be handled because it stayed frozen.

File: plugins.json [hunk 1]
```diff
@@ -833,4 +833,16 @@
         "UrlSourceCode": "https://github.com/brunolm/workflowy-wf",
         "IcoPath": "https://cdn.jsdelivr.net/gh/brunolm/workflowy-wf@v1.0.0/icon.jpg"
     }
+	{
+        "ID": "5A1E2AA6-1C6A-4130-990E-973FC5795D87",
```
Comment (jjw24) [inline plugins.json:837]: Missing a comma here :)

**Review by jjw24 — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (ameenaltajer) [reply plugins.json:837]: Oh thanks Jeremy, my apologies I must have forgotten about this :)

**Review by ameenaltajer — COMMENTED** [review]
Marked as reviewed.

**Comment by ameenaltajer** [comment]
> > 1. This will stay frozen until you finish the OAuth acceptance and it will go away since there is an HTTP server waiting for the callback response, did you finish the steps and it's still frozen? If you didn't please do.
> 
> Yeah ok that makes sense. I didn't finish the OAuth acceptance and I wonder if that's something that needs to be handled because it stayed frozen.

No problem, I agree too, I was planning to fix that later on but given that this hinders the experience a bit, let me fix it and keep you posted.

**Comment by ameenaltajer** [comment]
Hey @jjw24, version v1.1.0 is up with the fixes you recommend, hope that's good for the merge :)

File: plugins.json [hunk 2]
```diff
@@ -832,5 +832,17 @@
         "UrlDownload": "https://github.com/brunolm/workflowy-wf/releases/download/v1.0.0/Flow.Launcher.Plugin.WorkflowyWF.zip",
         "UrlSourceCode": "https://github.com/brunolm/workflowy-wf",
         "IcoPath": "https://cdn.jsdelivr.net/gh/brunolm/workflowy-wf@v1.0.0/icon.jpg"
+    },
+	{
```
Comment (jjw24) [inline plugins.json:836]:
```suggestion
    {
```

**Review by jjw24 — APPROVED** [review]
Great plugin, thanks for the submission.

I noticed you haven't used some of the methods available in Flow's API e.g. load/save settings, open browser, etc which may help save you some lines (you can see example usage in Flow's default plugins).

**Comment by ameenaltajer** [comment]
Thanks for the merge Jeremy, I'll refactor many of these items in the next couple of releases.

**Pull request merged**